site/content/user-guide/amqp-adapter.md

```diff
@@ -262,6 +261,36 @@ The table below provides an overview of the properties that must be set on a com
 | *correlation-id* | yes             | *properties*             | *string*     | MUST contain the value of the *correlation-id* property of the command request message. |
 | *status*         | yes             | *application-properties* | *integer*    | MUST contain a status code indicating the outcome of processing the command at the device (see [Command & Control API]({{< ref "/api/Command-And-Control-API.md" >}}) for details). |
 
+### Examples
+
+The AMQP adapter client can be used to simulate a device which receives commands and sends responses back to the application. The command line client is used to simulate an application sending commands to devices and receiving command responses from devices.
+
+Start the AMQP adapter client, as follows:
+
+    /hono/cli/target$ java -jar hono-cli-*-exec.jar --spring.profiles.active=amqp-command --hono.client.username=sensor1@DEFAULT_TENANT --hono.client.password=hono-secret
+    $ 04:13:49.651 [vert.x-eventloop-thread-0] INFO  o.e.hono.cli.CommandAndControlClient - Device is now ready to receive commands (Press Ctrl + c to terminate)
```

Comment:
IMHO this should be adapted to the real output, i.e. not a log statement anymore

Reply:
yes, but i i used a log statement in this PR. i will update it if i remove the log statement(as you mention previously)